tests/baselines/reference/docker/office-ui-fabric.log

```diff
@@ -221,32 +139,95 @@ lerna info Executing command in 42 packages: "yarn run build"
+office-ui-fabric-react: 2 import * as styles from './Layer.Example.scss';
+office-ui-fabric-react:                           ~~~~~~~~~~~~~~~~~~~~~~
+office-ui-fabric-react: src/components/Layer/examples/Layer.Hosted.Example.tsx:8:25 - error TS2307: Cannot find module './Layer.Example.scss'.
+office-ui-fabric-react: 8 import * as styles from './Layer.Example.scss';
+office-ui-fabric-react:                           ~~~~~~~~~~~~~~~~~~~~~~
+office-ui-fabric-react: src/components/SelectedItemsList/SelectedPeopleList/Items/EditingItem.tsx:11:31 - error TS2307: Cannot find module './EditingItem.scss'.
+office-ui-fabric-react: 11 import * as stylesImport from './EditingItem.scss';
+office-ui-fabric-react:                                  ~~~~~~~~~~~~~~~~~~~~
+office-ui-fabric-react: src/components/SelectedItemsList/SelectedPeopleList/Items/ExtendedSelectedItem.tsx:8:31 - error TS2307: Cannot find module './ExtendedSelectedItem.scss'.
+office-ui-fabric-react: 8 import * as stylesImport from './ExtendedSelectedItem.scss';
+office-ui-fabric-react:                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+office-ui-fabric-react: src/components/pickers/BasePicker.tsx:14:31 - error TS2307: Cannot find module './BasePicker.scss'.
+office-ui-fabric-react: 14 import * as stylesImport from './BasePicker.scss';
+office-ui-fabric-react:                                  ~~~~~~~~~~~~~~~~~~~
+office-ui-fabric-react: src/components/pickers/PeoplePicker/PeoplePickerItems/SelectedItemDefault.tsx:9:31 - error TS2307: Cannot find module './PickerItemsDefault.scss'.
+office-ui-fabric-react: 9 import * as stylesImport from './PickerItemsDefault.scss';
+office-ui-fabric-react:                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+office-ui-fabric-react: src/components/pickers/PeoplePicker/PeoplePickerItems/SelectedItemWithMenu.tsx:9:31 - error TS2307: Cannot find module './PickerItemsDefault.scss'.
+office-ui-fabric-react: 9 import * as stylesImport from './PickerItemsDefault.scss';
+office-ui-fabric-react:                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+office-ui-fabric-react: src/components/pickers/PeoplePicker/PeoplePickerItems/SuggestionItemDefault.tsx:7:31 - error TS2307: Cannot find module './SuggestionItemDefault.scss'.
+office-ui-fabric-react: 7 import * as stylesImport from './SuggestionItemDefault.scss';
+office-ui-fabric-react:                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+office-ui-fabric-react: src/components/pickers/Suggestions/Suggestions.tsx:13:31 - error TS2307: Cannot find module './Suggestions.scss'.
+office-ui-fabric-react: 13 import * as stylesImport from './Suggestions.scss';
+office-ui-fabric-react:                                  ~~~~~~~~~~~~~~~~~~~~
+office-ui-fabric-react: src/components/pickers/Suggestions/SuggestionsItem.tsx:8:31 - error TS2307: Cannot find module './Suggestions.scss'.
+office-ui-fabric-react: 8 import * as stylesImport from './Suggestions.scss';
```

Comment:
... did we somehow break all their `scss` imports? Is this somehow our fault? This build may have been silent for a few days due to a change in `npm` that made it error before it even produced a log, so the suspect time frame is a bit longer than just today.

@sandersn anything come to mind?

Reply:
Um, no? @sheetalkamat might know something about imports?

Otherwise maybe you could grab their most recent build and try it would 3.7 and the 3.8 beta to see if it's broken there too.